json_minify/lib/JSON_minify.pm

```diff
@@ -117,6 +123,10 @@ sub minify_string {
             my $leftcontext = substr($input_string, 0, $input_pos-1);
```

Comment:
Change the `0` here to `$prevIndex`.

Reply:
Ahh, that makes sense... but that's the string copy approach. I was looking to see if there was just a way to tell the regex to start from an index. I didn't see that here they're already copying the string.